models/bajas.go

```diff
@@ -0,0 +1,87 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/astaxie/beego/logs"
+	"github.com/astaxie/beego/orm"
+)
+
+type TrRevisionBaja struct {
+	Bajas         []int
+	Aprobacion    bool
+	Observaciones string
+}
+
+type FormatoBaja struct {
+	Consecutivo    string
+	Elementos      []int
+	FechaRevisionA string
+	FechaRevisionC string
+	Funcionario    int
+	Revisor        int
+}
+
+// PostRevisionComite hace la actualización de los movimientos de acuerdo a la revisión
+func PostRevisionComite(n *TrRevisionBaja) (ids []int, err error) {
+	o := orm.NewOrm()
+	err = o.Begin()
+
+	defer func() {
+		if r := recover(); r != nil {
+			o.Rollback()
+			logs.Error(r)
+		} else {
+			o.Commit()
+		}
```

Comment:
- Lanzar un `panic` adicionalmente
- Con lo anterior, se podría omitir el `else`
Algo así...
```suggestion
			logs.Error(r)
			panic(r)
		}
		o.Commit()
```